Test/Integration/TestCase.php

```diff
@@ -243,13 +247,10 @@ private function bootstrap()
      *
      * @return mixed method return
      */
-    protected function invokeMethod(&$object, $methodName, array $parameters = [])
+    protected function invokeMethod(object $object, string $methodName, array $parameters = [])
     {
         $reflection = new \ReflectionClass(get_class($object));
-        $method = $reflection->getMethod($methodName);
-        $method->setAccessible(true);
-
-        return $method->invokeArgs($object, $parameters);
+        $reflection->getMethod($methodName)->invokeArgs($object, $parameters);
```

Comment:
Oopsy, I missed this small issue, you need the method to return the result so that the `invokeMethod()` calls work in the QueueTest file.

Will re-add it ASAP.

Reply:
Ah! Good catch @damcou . My bad. I was rushing on this one. Thank you for fixing!